extensions/mssql/config.json

```diff
@@ -1,6 +1,6 @@
 {
 	"downloadUrl": "https://github.com/Microsoft/sqltoolsservice/releases/download/v{#version#}/microsoft.sqltools.servicelayer-{#fileName#}",
-	"version": "3.0.0-release.61",
+	"version": "3.0.0-release.64",
```

Comment:
Are there any other changes we need to be worried about here on the STS side, or does this just include the revert?

Reply:
This is just the revert + some minor non-code related changes